src/store/Modules/Connections.ts

```diff
@@ -98,9 +99,10 @@ export default class Connections extends VuexModule {
       this.context.commit("updateGlobalConnection", { data, connectionIdx })
     } catch (err) {
       console.log(err)
+      this.context.commit("updateGlobalConnection", {})
```

Comment:
Make sure you have added checks in the Mutation to handle empty objects.
Refer - `addNewProject()` Mutation.